br/pkg/storage/gcs.go

```diff
@@ -314,6 +324,8 @@ func (s *GCSStorage) Rename(ctx context.Context, oldFileName, newFileName string
 // used in tests
 var mustReportCredErr = false
 
+const gcsClientCnt = 16
```

Comment:
make it configurable?

Reply:
I'll leave it for future performance analysis.